### ISSUE TITLE
Maybe show ice from all zoom levels

### ISSUE THREAD

**Issue by jidanni** [der-stefan/OpenTopoMap#250]
https://opentopomap.org/#map=8/59.7121/-43.3521 cheery green
https://opentopomap.org/#map=9/59.9509/-42.8769 ice covered reality

**Comment by der-stefan** [comment]
Good catch. Perhaps that's why it is called Greenland. But there is also a lot of orange. Simple minds tend to go for their own color presumably.
